sdks/python/apache_beam/io/gcp/pubsub_test.py

```diff
@@ -109,8 +110,9 @@ def test_repr(self):
 class TestReadFromPubSubOverride(unittest.TestCase):
 
   def test_expand_with_topic(self):
-    p = TestPipeline()
-    p.options.view_as(StandardOptions).streaming = True
+    options = PipelineOptions([])
+    p = TestPipeline(options=options)
+    options.view_as(StandardOptions).streaming = True
```

Comment:
Technically this works, but setting `streaming` should happen before passing `options` to TestPipeline, since TestPipeline might make a copy of `options`, already evaluated `streaming` in the constructor, etc.